### PR TITLE
Support python 3.9

### PR DESCRIPTION
**Pull request by hhtong**

### CODE REVIEW

File: penaltymodel_mip/setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 install_requires = ['dimod>=0.6.0,<0.10.0',
                     'networkx>=2.0,<3.0',
-                    'ortools>=6.6.4659,<8.0.0',
```
Comment (arcondello) [inline penaltymodel_mip/setup.py:11]: it makes me a bit nervous to remove the upper bound entirely. They change their API semi often.

Comment (hhtong) [reply penaltymodel_mip/setup.py:11]: 👍  I can make it <8.1.0 ?

Comment (arcondello) [reply penaltymodel_mip/setup.py:11]: If an 8.x is working, then I think `<9.0.0`

Comment (hhtong) [reply penaltymodel_mip/setup.py:11]: Done

**Review by arcondello — APPROVED** [review]

File: penaltymodel_mip/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
-dimod==0.9.0
-networkx==2.0
-ortools>=6.6.4659,<8.0.0; platform_machine != "x86"
+dimod==0.9.11
+networkx==2.4
+ortools==8.0.8283; platform_machine != "x86" and python_version == "3.9"
+ortools>=6.6.4659,<8.0.0; platform_machine != "x86" and python_version < "3.9"
```
Comment (randomir) [inline penaltymodel_mip/requirements.txt:4]: Why not just:
```
ortools>=6.6.4659,<9.0.0; platform_machine != "x86"
```

Comment (hhtong) [reply penaltymodel_mip/requirements.txt:4]: ortools 8.0. dropped support for python 3.5

**Comment by hhtong** [comment]
Closes #125

**Comment by hhtong** [comment]
Closes #124

File: penaltymodel_mip/requirements.txt (hunk 2 above)
Comment (randomir) [reply penaltymodel_mip/requirements.txt:4]: Sure, but `ortools>=6.6.4659,<9.0.0` will install the best candidate for any environment.

Comment (hhtong) [reply penaltymodel_mip/requirements.txt:4]: Ah whoops, that's true. Will make this change now. Thanks!

**Pull request merged**